src/main/java/Duke.java

```diff
@@ -6,5 +8,72 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        // Greeting Message
+        System.out.println("Hello, I'm Duke!");
+        System.out.println("What can I do for you?");
+
+        Scanner sc = new Scanner(System.in);
+        int inputs = 0;
+        Task[] list = new Task[100];
+        while(true) {
+            String input = sc.next();
+            // Exit Command
+            if(input.equals("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            }
+            else {
+                // Check for other commands
+                switch(input) {
+                case ("list"):
+                    System.out.println("Here are the tasks in your list:");
+                    for(int i = 0;i < inputs; i++) {
+                        System.out.print((i+1) + ".");
+                        System.out.println(list[i].toString());
+                    }
+                    break;
+
+                case ("done"):
+                    int done = sc.nextInt();
+                    list[done - 1].markAsDone();
+                    System.out.println("Nice! I've marked this task as done:");
```

Comment:
printed messages could be extracted as constants to the top of the code for easy editing (and localisation if needed)